der_derive/src/sequence/field.rs

```diff
@@ -348,11 +348,11 @@ mod tests {
             quote! {
                 let implicit_field = ::der::asn1::ContextSpecific::<>::decode_implicit(
                         reader,
-                        ::der::TagNumber::N0
+                        ::der::TagNumber(0u32)
```

Comment:
Shouldn't these be...

```suggestion
                        ::der::TagNumber::new(0u32)
```

Reply:
We can't use fn in match

```
error[E0164]: expected tuple struct or tuple variant, found associated function `::der::TagNumber::new`
   --> der/tests/derive.rs:121:18
    |
121 |         #[derive(Choice, Debug, Eq, PartialEq)]
    |                  ^^^^^^ `fn` calls are not allowed in patterns
```